Britannia/common/culture/cultures/00_germanic.txt

```diff
@@ -116,52 +117,22 @@ saxon = {
 	}
 }
 
-oldfrisian = {
-	color = { 209 186 94 }
-	
-	ethos = ethos_stoic
-	heritage = heritage_west_germanic
-	language = language_anglic
-	martial_custom = martial_custom_male_only
-	traditions = {
-		tradition_things
-		tradition_seafaring
-		tradition_hird
-		tradition_concubines
-	}
-	
-	name_list = name_list_oldfrisian
-
-	coa_gfx = { frisian_coa_gfx western_coa_gfx } 
-	building_gfx = { western_building_gfx } 
-	clothing_gfx = { dde_hre_clothing_gfx western_clothing_gfx } 
-	unit_gfx = { western_unit_gfx } 		
-
-	ethnicities = {
-		25 = caucasian_blond
-		15 = caucasian_ginger
-		35 = caucasian_brown_hair
-		25 = caucasian_dark_hair
-	}
-}
-
 frisian = {
 	color = { 0.7 0.3 0.0 }
```

Comment:
I'm thinking about keeping the Old X culture colours for consistency with the CK2 mod, particularly for Frankish (who I think is a dark blue in vanilla).